### PR TITLE
feat: file of repos for fetching

### PR DESCRIPTION
**Pull request by Slug-Boi**
This PR lets the user use the repo-file flag to specify a file containing links to repo as well as branch and whitelist file information. This can be simplified by combining it with the branch and whitelist flags.

Resolves [#37](https://github.com/orgs/NovoNordisk-OpenSource/projects/2/views/3?pane=issue&itemId=56771516)

### CODE REVIEW

**Review by JonasSkjodt — APPROVED** [review]

**Review by Alex-itu — APPROVED** [review]

File: src/cmd/fetch.go [hunk 1]
```diff
@@ -87,6 +133,7 @@ var fetchCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(fetchCmd)
 
-	fetchCmd.Flags().String("branch", "", "Branch name for all repositories")
-	fetchCmd.Flags().String("whitelist", "", "Path to a whitelist file for all repositories")
+	rootCmd.PersistentFlags().String("branch", "", "Branch name for all repositories")
+	rootCmd.PersistentFlags().String("whitelist", "", "Path to a whitelist file for all repositories")
+	rootCmd.PersistentFlags().String("repo-file", "", "Path to a file containing the list of repositories to fetch")
```
Comment (August-Brandt) [inline src/cmd/fetch.go:138]: Why are we again using `PersistentFlags`?

Comment (Slug-Boi) [reply src/cmd/fetch.go:138]: I will cherry pick the commit in if you can find the SHA for me

Comment (Slug-Boi) [reply src/cmd/fetch.go]: I have cherry picked the commit :+1:

File: src/repos.txt
Comment (Haruyzal) [inline src/repos.txt:1]: Should this still be here?

Comment (Slug-Boi) [reply src/repos.txt:1]: This file has been removed

**Review by August-Brandt — APPROVED** [review]
Looks good now

**Review by Haruyzal — APPROVED** [review]
Nice removal. Huge W.

**Pull request merged**